### PR TITLE
[Utils] Modify memory usage script

### PR DESCRIPTION
**Pull request by jihochu**
It uses /proc/$PID/smaps information to check current memory usage.

Please refer doc:
https://man7.org/linux/man-pages/man5/proc.5.html

Signed-off-by: Jiho Chu <jiho.chu@samsung.com>

### CODE REVIEW

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Comment by SeoHyungjun** [comment]
LGTM!

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: Applications/utils/mem_usage.sh [hunk 1]
```diff
@@ -16,30 +16,28 @@ log="\nPID - $process_id :"
...
-	done
-	outlog="$outlog$count $mem_usage kB $cpu_output\n"
-	let mem_peak='mem_usage > mem_peak ? mem_usage : mem_peak'
-	sleep 1s # configure the sleep time if the program does not run long enough
-    else
-	break
-    fi
+	if [ -d "/proc/$process_id" ]; then
+		count=$((count + 1))
+		mem_usage=`cat /proc/$process_id/smaps | grep -E "Rss" | awk '{sum += $2;}END{print sum;}'`
```
Comment (jijoongmoon) [inline Applications/utils/mem_usage.sh:21]: Could you leave the comment why we change ```$process_is/status``` to ```smaps``` ??

**Review by songgot — APPROVED** [review]
LGTM

File: Applications/utils/mem_usage.sh (hunk 1 above)
Comment (jihochu) [reply Applications/utils/mem_usage.sh:22]: The commnet is added.

**Pull request merged**